hummingbot/templates/conf_simple_trade_strategy_TEMPLATE.yml

```diff
@@ -0,0 +1,31 @@
+########################################################
+###       Pure market making strategy config         ###
+########################################################
+
+# The following configuations are only required for the
+# pure market making strategy
+
+# Exchange and token parameters
+market: null
+
+# Type of Order (Limit/Market)
+order_type: null
+
+# Size of the Order
+order_amount: null
+
+# Price of Order (in case of Limit Order)
+order_price: null
+
+# Time in seconds before cancelling and rerunning strategy
+# If tick size is 60, the bot cancels active orders and reruns after a minute
+cancel_order_wait_time: null
```

Comment:
Is this used in this strategy?  Doesn't it just place a single order (limit or market) and that's it? No refreshing?

Reply:
it cancels the limit order after the wait time

Reply:
OK, then need to update the description since it doesn't rerun.

`For limit orders, time in seconds the order is cancelled after being placed if the order is not filled`.

Also probably shouldn't use "tick size" since that typically refers to a unit of price (and not other things such as time).  If the "value" is..